src/sass/_common/common.scss

```diff
@@ -440,7 +435,11 @@ label {
 
 #content-holder {
     flex: 1 1 auto;
-    margin-top: 25px;
+    margin-top: 48px;
+
+    @media(max-width: 880px){
```

Comment:
i think it should be 768 (from tablet breakpoint) @prince-deriv ?

Reply:
This is a specific breakpoint to handle the main content, since the header contents are dynamic we can only handle this on `768px` if we update the header.